spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SortOrderToSpark.java

```diff
@@ -19,44 +19,56 @@
 
 package org.apache.iceberg.spark;
 
+import java.util.Map;
 import org.apache.iceberg.NullOrder;
 import org.apache.iceberg.SortDirection;
 import org.apache.iceberg.transforms.SortOrderVisitor;
 
 class SortOrderToSpark implements SortOrderVisitor<OrderField> {
+
+  private final Map<Integer, String> quotedNameById;
+
+  SortOrderToSpark(Map<Integer, String> quotedNameById) {
+    this.quotedNameById = quotedNameById;
```

Comment:
Nit: do you think its slightly easier to reason if we have SortOrderToSpark constructor take the schema, and then make the map inside constructor?  Just took me a little bit to find where this map was being populated.

Reply:
I did admittedly on first read through have that same wondering.

Reply:
I would probably make this accept a `Schema` instead. I agree that it would be easier to follow.

Reply:
Will fix.

Reply:
Done.